src/main/java/org/folio/consortia/service/impl/UserTenantServiceImpl.java

```diff
@@ -52,14 +74,40 @@ public UserTenant getById(UUID consortiumId, UUID id) {
     return converter.convert(userTenantEntity, UserTenant.class);
   }
 
+  @Override
+  public UserTenant save(UUID consortiumId, UserTenant userTenantDto) {
+    String currentTenantId = HelperUtils.getTenantId(folioExecutionContext);
+    consortiumService.checkConsortiumExistsOrThrow(consortiumId);
+
+    Optional<UserTenantEntity> userTenant = userTenantRepository.findByUserIdAndTenantId(userTenantDto.getUserId(), IS_PRIMARY_TRUE);
```

Comment:
Lets find current tenantId from user that has primary affiliation, in this case we competely sure that using correct tenantId of the real user